Description: systemd: Cleanup ntp-wait.service
 It seems more proper to do WantedBy=time-sync.target, and it achieves the
 proper effect.  Then, WantedBy implies a reverse Before, so the
 Before=time-sync.target is redundant.
 .
 I assume the original arrangement was copied from systemd-timesyncd.  That
 is more analogous to ntpd, not ntpwait.  In fact, systemd needs to add a
 systemd-timesyncd-wait (which would be analogous to ntp-wait):
 https://github.com/systemd/systemd/issues/5097
Forwarded: https://gitlab.com/NTPsec/ntpsec/merge_requests/581
Author: Richard Laager <rlaager@wiktel.com>
Last-Update: 2017-11-26
--- a/etc/ntp-wait.service
+++ b/etc/ntp-wait.service
@@ -3,8 +3,6 @@
 Requisite=ntpd.service
 After=ntpd.service
 Conflicts=systemd-timesyncd.service
-Wants=time-sync.target
-Before=time-sync.target
 ConditionVirtualization=!container
 ConditionCapability=CAP_SYS_TIME
 
@@ -15,4 +13,4 @@
 StandardOutput=null
 
 [Install]
-WantedBy=multi-user.target
+WantedBy=time-sync.target
